src/main/java/uk/gov/justice/digital/oasys/jpa/entity/Offender.java

```diff
@@ -77,21 +78,21 @@
     @Column(name = "MERGE_PNC_NUMBER")
     private String mergePncNumber;
     @Column(name = "DATE_OF_DEATH")
-    private Timestamp dateOfDeath;
+    private LocalDate dateOfDeath;
```

Comment:
Double check these are just dates in the database and not DateTimes. If it's a date time convert in the from method.

Reply:
They are Dates